server/helpers/adapters_tracker.go

```diff
@@ -119,11 +125,99 @@ func (a *AdaptersTracker) DeployAdapter(ctx context.Context, adapter models.Adap
 			return ErrAdapterAdministration(err)
 		}
 
+	case "kubernetes":
+		kubeClient, err := meshkitkube.New([]byte(""))
```

Comment:
@Azanul we're using the helm client, yes? Ideally, we're using the exact same mechanism to deploy Meshery Adapters as we are to deploy Meshery Operator.

Reply:
kubernetes client not helm